sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
I see, until that's fix by GH then, if you don't mind uploading it when you get a chance that would be great. Thank you!